plutus-core/generators/Language/PlutusCore/Generators/Interesting.hs

```diff
@@ -243,8 +246,8 @@ genDivideByZeroDrop = do
     TermOf i iv <- genTermLoose typedInt
     let term =
             mkIterApp () (mkIterInst () Function.const [int, int])
-                [ mkIterApp () (Builtin () op) [i, mkConstant @Integer () 0]
-                , mkConstant @Integer () iv
+                [ mkConstant @Integer () iv
+                , mkIterApp () (Builtin () op) [i, mkConstant @Integer () 0]
```

Comment:
So apparently we do have a strictness tests, but I messed up the order of arguments and the test was dysfunctional. That fixes it.